src/main/connector/system/Windows/Windows.yaml

```diff
@@ -267,8 +256,8 @@ monitors:
         attributes:
           id: $1
         metrics:
-          system.disk.io{disk.io.direction="read"}: $2
-          system.disk.io{disk.io.direction="write"}: $3
+          system.disk.io{direction="read"}: $2

```

Comment:
`system.disk.io` is correct. Please revert. (see https://opentelemetry.io/docs/specs/semconv/system/system-metrics/#metric-systemdiskio)